src/lib/iomgr.cpp

```diff
@@ -153,7 +153,15 @@ void IOManager::start(const iomgr_params& params, const thread_state_notifier_t&
 
     // Start all reactor threads
     set_state(iomgr_state::reactor_init);
-    create_worker_reactors();
+
+    uint32_t num_fibers_per_worker = 0;
```

Comment:
How about a form of where we don't have to explicitly type `uint32_t`? This will be a code smell in Sonar.
```
create_worker_reactors((0 < params.num_fibers) ? params.num_fibers : IM_DYNAMIC_CONFIG(thread.num_fibers));
```